src/inc/til/point.h

```diff
@@ -33,6 +33,12 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             THROW_HR_IF(E_ABORT, !base::MakeCheckedNum(y).AssignIfValid(&_y));
         }
 
+        template<typename TilMath, typename T>
+        constexpr point(TilMath, T x, T y, std::enable_if_t<std::is_floating_point_v<decltype(std::declval<T>())>, int> /*sentinel*/ = 0) :
+            point(, TilMath::template cast<ptrdiff_t>(y))

```

Comment:
it's remarkable that this compiles

Reply:
or does what it is supposed to

Reply:
I think it's unused so it's probably not compiling. I think I meant to trash that.